{{ cookiecutter.project_name }}/.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,42 +1,42 @@
-name: Сообщение об ошибке
-description: Что то работает не правильно?
+name: Bug Report
+description: Something work wrong?
 labels: "needs triage,bug"
 
 body:
   - type: textarea
     id: description
     attributes:
-      label: Описание
+      label: Description
       description: >-
-        Понятное и чистое описание того что представляет из себя баг.
+        A detailed and clear description of what the bug is.
     validations:
       required: true
 
   - type: textarea
     id: expected
     attributes:
-      label: Что ожидалось?
+      label: What expected?
       description: >-
-        Понятное и чистое описание того что вы ожидали чтобы произошло.
+        A detailed and clear description of what you expected to happen.
 
   - type: input
     id: commit_repo
     attributes:
-      label: Версия репозитория
+      label: Repository's commit
       placeholder: cd {{ cookiecutter.project_name }}; git rev-parse HEAD
     validations:
       required: true
   - type: input
     id: python_version
     attributes:
-      label: Версия Python
+      label: Python Version
```

Comment:
Same as previous. Don't use capitals!

Reply:
But why?

Reply:
Because it doesn't look good. Why the hell is Version capitalized? That's not according to the rules.